src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -1,6 +1,7 @@
 package seedu.addressbook.commands;
 
 import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.Printable;
```

Comment:
should arrange alphabetical order here. In really huge projects, if you didn't arrange it nicely, its likely that at some point of time you'll have duplicate import statements which can be unsightly :P